modules/core/src/commonMain/kotlin/fr/acinq/lightning/Features.kt

```diff
@@ -154,17 +154,6 @@ sealed class Feature {
         override val scopes: Set<FeatureScope> get() = setOf(FeatureScope.Invoice)
     }
 
-    // We historically used the following feature bit in our invoices.
-    // However, the spec assigned the same feature bit to `option_scid_alias` (https://github.com/lightning/bolts/pull/910).
-    // We're moving this feature bit to 148, but we have to keep supporting it until enough wallet users have migrated, then we can remove it.
-    // We cannot rename that object otherwise we will not be able to read old serialized data.
-    @Serializable
-    object TrampolinePayment : Feature() {
```

Comment:
I verified that:
- those features are not permanent and thus haven't been written in `channelFeatures`
- the init features written to `localParams.features` were always defined as `optional`